### PR TITLE
Add addPrimaryKeyConstraint for AlterTableBuilder

### LINKED ISSUE #636: Changing the primary key of existing tables
Seems like `AlterTableBuilder` currently misses `addPrimaryKey` method?

### PR DESCRIPTION
**Pull request by n7olkachev**
Closes #636

### CODE REVIEW

File: test/node/src/schema.test.ts [hunk 1]
```diff
@@ -2427,6 +2427,28 @@ for (const dialect of DIALECTS) {
...
+              mysql: {
+                sql: 'alter table `test` add constraint `test_pkey` primary key (`integer_col`)',
+                parameters: [],
+              },
+              sqlite: NOT_SUPPORTED,
+            })
+
+            await builder.execute()
+          })
+        })
```
Comment (igalklebanov) [inline test/node/src/schema.test.ts:2450]: nitpicky

let's use a separate `if (dialect !== 'sqlite')` block for this.

**Review by igalklebanov — APPROVED** [review]
Thank you! 🙏 

LGTM! 🚀  
left a minor comment..

File: test/node/src/schema.test.ts (hunk 1 above)
Comment (igalklebanov) [reply test/node/src/schema.test.ts:2450]: let's add a test case for 2 columns.

File: test/node/src/schema.test.ts [hunk 2]
```diff
@@ -2427,6 +2427,28 @@ for (const dialect of DIALECTS) {
           })
         })
 
+        describe('add primary key constraint', async () => {
```
Comment (igalklebanov) [inline test/node/src/schema.test.ts:2430]: let's add a test case with 2 columns.

File: test/node/src/schema.test.ts (hunk 1 above)
Comment (n7olkachev) [reply test/node/src/schema.test.ts]: done

File: test/node/src/schema.test.ts (hunk 2 above)
Comment (n7olkachev) [reply test/node/src/schema.test.ts]: done

**Comment by koskimas** [comment]
Thank you!

**Pull request merged**